_pages/Housing_no_basecamp.md

```diff
@@ -33,10 +33,25 @@ It is also possible to ask the administration in your centre
 to send an email to employees to see if anyone knows of a spare room.
 
 
-### Link to useful documents
+### Link to useful websites
 
 - [IHCPH](https://ihcph.kk.dk/indhold/documentation-requirements-and-examples)
 
 - [Boligportal](https://www.boligportal.dk/?gclid=Cj0KCQiAuefvBRDXARIsAFEOQ9H5daFK1q-W6NFZlCxCCRUz_Ac_Z9IC6Els5Vjem0ZXNJKnEie8UpAaAiZ1EALw_wcB)
 
+- [lejebolig](https://www.lejebolig.dk/)
 
+## Advice for searching rental apartments
+
+Many of the CPH Biosciences students found an appartment through boligportal. While many locals don’t rent through such websites, but either buy a whole apartment or buy shares in so called "andelbolig". The first is not ideal due to the investment cost and uncertain future after the PhD, while the latter is foreigner unfriendly since it requires some years in waiting lists. 
+
+Employers often have login credentials that can be used to avoid paying the fees required for the housing seekers (e.g. CPR has an account). Apartments will mostly be for 1 to 3 people, but exeptionally you will find some for more. Many previous students have opted to rent something together with fellow students. If you use one of these accounts that is not coupled to your own email, you can create a second acount on a private email with a search agent that would notify of any new offers matching your specific criteria. You can then contact them through the employers account. Write a short but personal message that you maybe personalize a bit every time and send to reply to offers. If you want to get an example, don't hesitate to ask older students.
```

Comment:
```suggestion
Employers often have login credentials that can be used to avoid paying the fees required for the housing seekers (e.g. CPR has an account). Apartments will mostly be for 1 to 3 people, but exceptionally you will find some for more. Many previous students have opted to rent something together with fellow students. If you use one of these accounts that is not coupled to your own email, you can create a second account on a private email with a search agent that would notify of any new offers matching your specific criteria. You can then contact them through the employers account. Write a short but personal message that you maybe personalize a bit every time and send to reply to offers. If you want to get an example, don't hesitate to ask older students.
```